### PR TITLE
CLA Request

### PR DESCRIPTION
**Pull request by AceHack**
Aleph.ai CLA Request

### CODE REVIEW

**Comment by nanis94** [comment]
micrasoft:main

**Comment by nanis94** [comment]
AlephZ-ai:main

**Comment by nanis94** [comment]
AlephZ-ai:main micrasoft:main

**Review by nanis94 — COMMENTED** [review]
AlephZ-ai:main micrasoft:main

**Comment by 1983Thai** [comment]
26a1b8a5dfc4509ea43c67b25771cf13e898e5ca

File: .github/FUNDING.yml
Comment (1983Thai) [inline .github/FUNDING.yml:1]: 26a1b8a5dfc4509ea43c67b25771cf13e898e5ca

**Review by Bysea0402 — APPROVED** [review]

File: src/ContributorLicenseAgreement.Core/Handlers/Helpers/ClaHelper.cs [hunk 1]
```diff
@@ -60,17 +64,19 @@ internal static string GenerateRetrievalKey(string gitHubUser, string claLink)
             return GenerateKey(user, claLink);
         }
 
-        internal static bool NeedsLicense(Cla primitive, PullRequest pullRequest)
+        internal static bool NeedsLicense(Cla primitive, PullRequest pullRequest, string origin)
```
Comment (Bysea0402) [inline src/ContributorLicenseAgreement.Core/Handlers/Helpers/ClaHelper.cs:67]: Need github license change.

**Review by Bysea0402 — CHANGES_REQUESTED** [review]
Github client change.

**Review by Teomelero — COMMENTED** [review]
DELETE

**Review by Dilekis — CHANGES_REQUESTED** [review]
@

**Pull request closed without merging**